### PR TITLE
Refactor/summary stats to use client side fetch

### PR DESCRIPTION
**Pull request by mrkarimoff**
This PR refactors `SummaryStatistics.tsx` component to use client-side data fetching with a Skeleton loading state and removes all `relalidateTag` functions used for revalidating summary data cache

### CODE REVIEW

File: app/(dashboard)/dashboard/_components/SummaryStatistics/SummaryStatistics.tsx [hunk 1]
```diff
@@ -1,56 +1,78 @@
...
-          title="Protocols"
-          initialData={protocolCount}
-          icon={<ProtocolIcon />}
-        />
+        {isProtocolStatsLoading ? (
+          <StatCardSkeleton title="Protocols" icon={<ProtocolIcon />} />
+        ) : (
+          <StatCard
+            title="Protocols"
+            initialData={protocolCount!}
```
Comment (jthrilly) [inline app/(dashboard)/dashboard/_components/SummaryStatistics/SummaryStatistics.tsx:31]: Let's rename this prop to just `data` or `value` now that it isn't being used as initialData.

**Review by jthrilly — CHANGES_REQUESTED** [review]
Looks good to me. Just the one change and I think it can be merged.

**Comment by jthrilly** [comment]
Oh, and one other small change (not really related to this PR, but should be fine to add here) - could you add the rounded-input class to the Skeleton component? It looks weird with them all being square

File: app/(dashboard)/dashboard/_components/SummaryStatistics/SummaryStatistics.tsx (hunk 1 above)
Comment (mrkarimoff) [reply app/(dashboard)/dashboard/_components/SummaryStatistics/SummaryStatistics.tsx]: Done here: https://github.com/complexdatacollective/Fresco/pull/115/commits/518153d7287f254ccaa45993d9e566b8ed8496a6

**Comment by mrkarimoff** [comment]
> Oh, and one other small change (not really related to this PR, but should be fine to add here) - could you add the rounded-input class to the Skeleton component? It looks weird with them all being square

Added here: https://github.com/complexdatacollective/Fresco/pull/115/commits/cc7b55688a040ac885076c9c5a2cbdc86ec113a5

**Review by jthrilly — APPROVED** [review]
Looks great!

**Pull request merged**